signerv2/remote_signer_client.go

```diff
@@ -0,0 +1,107 @@
+package signerv2
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/Layr-Labs/eigensdk-go/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+type RpcRequest struct {
```

Comment:
7a6df19465ed37185f3719c841bd4ec3d2e9ddaf
I didn't include 2 in the request since it's the first time we are using it and we never used `1.0` - also added a comment.